tests/test_forward_ref.py

```diff
@@ -505,6 +505,52 @@ class What(BaseModel):
     assert m.base.literal == 1
 
 
+@skip_pre_37
+def test_postponed_annotations_annotated_alias(create_module):
+    module = create_module(
+        # language=Python
+        """
+from __future__ import annotations
+from typing import Annotated
+from pydantic import BaseModel, Field
+
+class What(BaseModel):
```

Comment:
please can we use a less confusing name than "what". "Foo", "Bar" and fruit names are commonly used, but not pronouns like "what"